### PR TITLE
Alert backend test

### PR DESCRIPTION
**Pull request by STEPHINRACHEL**

### CODE REVIEW

File: backend-admin/test/alerts.js [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+const openshiftAdminPassword = process.env.OPENSHIFT_ADMIN_PASSWORD;
+describe('MDC alerts test', function() {
+    this.timeout(0);
+    before('authenticate with oauth proxy', async() => {
+        await init();
+        mdcNamespace = await getNamespaces("mobile-developer-console");
+        await getParam();
+    });
+    it('should check whether MDC alerts are already active', async ()=>{
+        await waitFor(async() => {
```
Comment (jhellar) [inline backend-admin/test/alerts.js]: Do we need to wait here? Would be enough to just check that no MDC alerts are active?

File: backend-admin/test/alerts.js [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            });
+            if((result.data.data.alerts.findIndex(alert =>
+                alert.labels.alertname === 'MobileDeveloperConsoleOperatorDown'
+            ))===-1){
+                return true;
+            }
+        }, 3 * 60000);
+    })
+});
+describe('MSS Alert test', function() {
```
Comment (jhellar) [inline backend-admin/test/alerts.js]: For MSS should we also scale the deployments back up and check that the alerts got deactivated (like for MDC and UPS)?

Comment (STEPHINRACHEL) [reply backend-admin/test/alerts.js]: MSS operator is behaving different from other operators, the operator maintains the state that's specified in the CR. In order to scale up/down other deployments, we need to scale up/down MSS operator first. After scaling up/down MSS operator we can check for the MSS alerts to disappear/appear. 
 https://issues.jboss.org/browse/AGMS-1155

File: backend-admin/test/alerts.js [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+    headers = await getHeaderWithOauthProxyCookie(alertUrl,openshiftAdminUser,openshiftAdminPassword);
+}
+// const getNameSpaces = async nameSpaceType => {
+//     const namespaces = await resource(TYPE.PROJECT, ACTION.GET_ALL);
+//     for (const ns of namespaces.items) {
+//         if (ns.metadata.name.endsWith(nameSpaceType)) {
+//             return ns.metadata.name;
+//         }
+//     }
+// }
```
Comment (psturc) [inline backend-admin/test/alerts.js]: This part is probably not needed anymore?

File: backend-admin/test/alerts.js [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+    })
+    it('should trigger mdc down alert', async() => {
+        await exec(`oc scale --replicas=0 dc mdc -n ${mdcNamespace}`);
+        await waitFor(async() => {
+            const result = await axios.request({
+                url: alertUrl,
+                headers,
+                method: 'get'
+            });
+            return result.data.data.alerts.find(alert =>
```
Comment (psturc) [inline backend-admin/test/alerts.js]: This condition in similar form is repeated couple of times in this file - would it be worth to make a function from it?

**Pull request closed without merging**